wbooks-api/Gemfile

```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.5.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
A few things about this (not corrections)
- When starting a new project, always try to remove gems you won't need. For example, if you're developing a pure API with no frontend at all (which is usually the case) you won't need things like turbolinks. Don't worry about this right now though, you can't know which ones of these will be useful or not.
-  When adding new gems, always add a comment before the gem so the next person knows what the gems do (you can copy paste the github definition or add a line yourself). Again, not a correction because you aren't really adding anything right now.